test/cgroups.bats

```diff
@@ -40,7 +40,7 @@ function teardown() {
 }
 
 @test "conmon custom cgroup" {
-	if [[ "$TRAVIS" == "true" ]]; then
+	if [[ "$CI" == "true" ]]; then
 		skip "travis container tests don't support systemd cgroups"
```

Comment:
Is this still true? If so, should `travis` be changed to 'CircleCI`?

Same comments apply below (to ctr.bats)

Reply:
Definitely, I will adapt it.